docs/DEPLOYMENT.md

```diff
@@ -1,7 +1,3 @@
-[![Version](https://img.shields.io/badge/version-development-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
```

Comment:
Might some people still look at the docs on GitHub instead of on GitBook? Maybe it is still useful to include the version badge here?

Reply:
Agreed. It looks _so so_ on GitBook but agree it's probably worth it for people reading the docs on Github. Added it back!